packages/api/src/schema/schema.yaml

```diff
@@ -1795,6 +2174,42 @@ paths:
               schema:
                 $ref: "#/components/schemas/error"
 
+  /vod-asset:
+    tags:
+      - vod-asset
+    get:
+      description: gets a list of vod assets defined by the user
+      responses:
+        200:
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/vod-asset"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+    post:
+      description: creates a new vod asset
+      responses:
+        201:
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/vod-asset"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
```

Comment:
I think we won't have a raw "create" like this. You can only create an asset from an import task or by uploading a new file. Uploading a file might require a little protocol like calling the API, receiving a pre-signed S3 URL to upload the file, then creating the `asset` with the link/pre-signed for that S3 file. So we might want to have separate/explicit API endpoints like `POST /assets/upload` perhaps which returns the S3 link, then another call which will basically import that file to the regular object store? The "import" would be only so that we run the `ffprobe` on the file and fill the asset spec, but could avoid the actual download&upload from S3 but only renaming the file which would already be in our own bucket. WDYT?